config/zookeeper_persister.go

```diff
@@ -197,6 +204,11 @@ func (z *ZkConfigPersister) ReadPersistedConfig() (io.Reader, error) {
 }
 
 func (z *ZkConfigPersister) currentConfigEventListener() (<-chan zk.Event, error) {
+	if z.initialized {
```

Comment:
This feels a little weird. The only thing we're doing here is printing a different log message. Did you mean to include some more in either of the branches? Same below.

Reply:
Just logging for now, but I will be using it for other things later.